requirements.txt

```diff
@@ -8,23 +7,25 @@ coveralls==1.11.1
 dj-database-url==0.5.0
 Django==3.0.3
 django-heroku==0.3.1
+django-timezone-field==4.0
 djangorestframework==3.11.0
+djangorestframework-simplejwt==4.4.0
 docopt==0.6.2
 entrypoints==0.3
 flake8==3.7.9
 gunicorn==20.0.4
+httpie==2.1.0
```

Comment:
please remove this, we don't need to test through the terminal when we are already using postman